code/modules/mob/living/simple_animal/borer/egg.dm

```diff
@@ -92,3 +92,7 @@
 		return
 	else
 		..()
+
+/obj/item/weapon/reagent_containers/food/snacks/borer_egg/Destroy()
```

Comment:
You don't have a `..()` at the end, which seems potentially mildly catastrophic on a reagent container